runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -26,26 +26,35 @@
 
 import com.google.common.collect.ImmutableSet;
 import javax.inject.Inject;
+
+import com.google.inject.Provides;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
+import net.runelite.api.*;
+
 import static net.runelite.api.ItemID.TEAK_CHEST;
-import net.runelite.api.VarPlayer;
-import net.runelite.api.Varbits;
+
```

Comment:
Remove these empty lines in the import section. You can tell your IDE to not do this as well.